packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -517,65 +513,56 @@ exports[`Podcast Promo Examples basic example 1`] = `
             width="100%"
           />
         </div>
-      </figure>
+      </div>
       <div
-        class="emotion-26 emotion-27"
+        class="emotion-24 emotion-25"
       >
         <h3
-          class="emotion-16 emotion-17"
+          class="emotion-14 emotion-15"
         >
           <a
-            class="emotion-14 emotion-15"
+            class="emotion-12 emotion-13"
             href="https://www.bbc.com/russian/media-47937790"
           >
             <span
```

Comment:
do we need this span?

Reply:
Yes I think. so - it's receiving a className that conditionally styles the text depending on whether the link has been visited, focused or hovered.